src/main/java/uk/gov/justice/digital/client/kinesis/KinesisReader.java

```diff
@@ -12,7 +12,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import scala.reflect.ClassTag$;
```

Comment:
why do we have scala reflect package? no alternatives in java?

Reply:
Since this is a scala API we're calling, no.